app/controllers/feedbacks_controller.rb

```diff
@@ -23,6 +23,7 @@ def create
   private
 
   def feedback_params
-    params.require(:feedback).permit(:referrer, :text, :email, :user_agent, :prison)
+    params.require(:feedback).permit(
+      :referrer, :text, :email, :user_agent, :prison)
```

Comment:
Either putting the `)` on a newline or putting the whole `.permit(…)` on its own line would be better, I think.